core/java/iesi-core/src/main/java/io/metadew/iesi/script/execution/instruction/data/text/TextReplace.java

```diff
@@ -3,29 +3,41 @@
 import io.metadew.iesi.script.execution.instruction.data.DataInstruction;
 
 import java.text.MessageFormat;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 public class TextReplace implements DataInstruction {
 
+    private final static String FIRST_OPERATOR = "text";
+    private final static String SECOND_OPERATOR = "start";
+    private final static String THIRD_OPERATOR = "end";
```

Comment:
replace with `replacement_char`

Reply:
We can not use name with underscore('_') otherwise it will throw Invalid group name, hence we will replace with 'replacementChar'.